src/constants/tokens.ts

```diff
@@ -1,5 +1,7 @@
 import type { BaseTokenInfo } from "../types/base"
 
+export const NATIVE_TOKEN_ADDRESS = "0x0000000000000000000000000000000000000000"
```

Comment:
It's exclusively for AuroraEngine SiloToSilo contract, I suggest putting in place where it's used.